corehq/apps/app_manager/views/utils.py

```diff
@@ -291,16 +291,20 @@ def handle_custom_icon_edits(request, form_or_module, lang):
 
 def update_linked_app_and_notify(domain, app_id, user_id, email):
     app = get_current_app(domain, app_id)
+    subject = _("Update Status for linked app %s") % app.name
```

Comment:
nit / question: do we have any favor/guidelines between using `string.format` and the `%` syntax? i thought `.format` was considered to be more of the standard these days, though am sure we have both all over the codebase.

Reply:
also seeing that you just moved this line which probably explains the `%`

Reply:
Yeah, I just moved it. For translated strings I also try to be cognizant of the fact that someone has to re-translate them whenever I change them, so I try not to make style-only changes. Not sure how much that matters, but that's my mental model of it.